submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <!--header-->
+    <header class="container">
+      <!--header-menu-->
+      <nav class="menu">
+        <ul class="menu">
+          <li class="menu-item"><a class="menu-item" href="#">Mail</a></li>
+          <li class="menu-item"><a class="menu-item" href="#">Images</a></li>
```

Comment:
Fix `outline` on focus state. Same problem on icons inside the popup.
![image](https://user-images.githubusercontent.com/16196199/102263055-881a6f00-3f1c-11eb-8d11-e1be26d6aa96.png)